src/App.tsx

```diff
@@ -225,6 +227,17 @@ function App() {
         <Route exact path="/recommendation">
           <RecommendationForm />
         </Route>
+
+        <Route path="/shopCart">
+          <ShopCart
+            cart={cart}
```

Comment:
Zastanawiam się, czy zamiast listy propsów można użyć ...useShopCart(); Co prawda będzie o jednego propsa za dużo, ale to nie powinno stanowić większego problemu.